common/lib/safe_lxml/safe_lxml/etree.py

```diff
@@ -9,6 +9,7 @@
 
 from lxml.etree import *  # pylint: disable=wildcard-import, unused-wildcard-import
 from lxml.etree import XMLParser as _XMLParser
+from lxml.etree import _ElementTree  # pylint: disable=unused-import
```

Comment:
Why is this needed? It's unused here, and I don't see `_ElementTree` mentioned elsewhere in the tree.